src/explore-education-statistics-admin/src/pages/release/datablocks/components/ChartConfiguration.tsx

```diff
@@ -234,8 +234,8 @@ const ChartConfiguration = ({
               name="alt"
               label="Alt text"
               hint="Brief and accurate description of the chart in less than 125 characters. Should not repeat the title."
-              maxLength={125}
-              rows={2}
+              maxLength={300}
```

Comment:
Can you test this on an actual screenreader and see if it actually reads out the entire alt for 300 chars?

Reply:
Just tested this in JAWS and everything was read out fine